lib/internal/util.js

```diff
@@ -186,4 +197,27 @@ exports.convertToValidSignal = function convertToValidSignal(signal) {
   }
 
   throw new Error('Unknown signal: ' + signal);
+}
+
+module.exports = {
```

Comment:
One problem with having it at the top is that you can run into variable definition issues, where you're exporting something that gets defined/set later on in the file. Sometimes you might be able to just pull the definition to the top, but other times the assignment may not be a simple one-liner. So putting it at the bottom means you never have to worry about such issues and doing it the same everywhere is good for consistency.